exercises/twelve-days/src/test/java/TwelveDaysTest.java

```diff
@@ -120,18 +120,21 @@ public void testVerseTwelve() {
     @Ignore("Remove to run test")
     @Test
     public void testMultipleVerses() {
+/*
```

Comment:
The more I think about this strategy, the more convinced I am that the net effect is more confusing than useful. I think that's because:

- only a few exercises require more than one method;
- that tests are commented out is potentially really non-obvious to users.

IMO it would be simpler to say: full API stubs for exercises 1-20: no stubs or class only stubs for exercises 21+ (with exceptions for really tricky signatures as in list-ops).

Thoughts, @FridaTveit @Smarticles101?